react-frontend/src/screens/components/Carousel/Carousel.jsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import Slider from 'react-slick';
+import ubuntu from '../../../images/ubuntu.png'; 
+import opensuse from '../../../images/opensuse.png'; 
+import debian from '../../../images/debian.png';
+import fedora from '../../../images/fedora.png';
+import rocky from '../../../images/rocky.png';
+import redhat from '../../../images/redhat.png';
+import clefos from '../../../images/clefos.png';
+import zos from '../../../images/zos.png'; 
+import almalinux from '../../../images/almalinux.png';
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+import './Carousel.css';
+
+function Carousel() {
+  const settings = {
+    dots: false,
+    infinite: true,
+    speed: 2000, // Slow down the speed for a smoother effect
```

Comment:
The carousel behaves a bit weirdly, speed sometimes goes fast and suddenly turns slow, also changing directions. Maybe we could check here?

Reply:
Could you please specify when you're encountering this issue? The carousel stops when you hover over it. If you could share a video demonstrating the problem, it would help understand it better.

Reply:
https://github.com/openmainframeproject/software-discovery-tool/assets/120312681/9e94b1f0-e3cd-47b4-818c-7c81803578d6

-  There's some kind of lag after every 1-2 seconds in the carousel
-  On changing tabs, it behaves abnormally